charts/logging-operator/README.md

```diff
@@ -26,6 +26,10 @@ To install the chart with the release name `my-release`:
 $ helm install --name my-release banzaicloud-stable/logging-operator
 ```
 
+### CRDs
+The CRDs are provisioned using crd-install hooks, rather than relying on a separate chart installation.
+If you already have these CRDs provisioned and don't want to remove them, you can disable the CRD creation by these hooks by passing `createCustomResource=false` (not required if using Helm v3).
```

Comment:
```suggestion
Use `createCustomResource=false` with Helm v3 to avoid trying to create CRDs from the `crds` folder and from templates at the same time.
```